web/kratos/README.md

```diff
@@ -97,13 +97,7 @@ npm run dev
 ## Create an Identity
 
 To add new user in the Kratos serice you have to expose the admin port, for this
-comment out the port forwarding "`9434:4434`" line in `kratos.yaml`
-
-- Generate recovery link in sh:
-
-```
-sh create_default_user.sh <your_custom_email@email.com>
-```
+comment out the port forwarding "`4434:4434`" line in `kratos.yaml`
```

Comment:
with registration this isn't necessary at all

Reply:
You may remove the create an identity section entirely

Reply:
I'll leave it here right now. We'll need to refactor the whole `README.md` in the near future.